readthedocs/projects/views/private.py

```diff
@@ -80,9 +81,24 @@ class ProjectDashboard(PrivateViewMixin, ListView):
     model = Project
     template_name = 'projects/project_dashboard.html'
 
+    def email_notification(self, request):
```

Comment:
Please, add a docstring here explaining what this method does.

Reply:
Added.